build.default.properties

```diff
@@ -28,6 +28,7 @@ build.type  = compass
 # Drupal
 drupal.modules.dir = ${project.basedir}/docroot/modules/custom
 drupal.theme.dir   = ${project.basedir}/docroot/themes/custom/myproject_responsive
+tests.enabled = false
```

Comment:
If this doesn't exist, assume no failures in phing-drupal-tests ?

Reply:
Thats right - if "tests.enabled" doesn't exist the tests just don't run.

Reply:
Perfect :)